src/libraries/System.CodeDom/tests/System/CodeDom/Compiler/CSharpCodeGenerationTests.cs

```diff
@@ -3485,5 +3485,36 @@ public void SpecialNewLineCharactersInStringsDoneCorrectly()
                       public static string StringWithSpecialNewLines = ""\u0085\u2028\u2029"";
                   }");
         }
+
+        [Fact]
```

Comment:
You're going to need to annotate these new tests as `[SkipOnTargetFramework(...)]` for .NET Framework, as it doesn't have the changes you're making.

Reply:
Done